hydroshare/settings.py

```diff
@@ -718,3 +722,161 @@
+IRODS_HOME_COLLECTION = '/hydrotest41Zone/home/hsproxy'
+IRODS_CWD = '/hydrotest41Zone/home/hsproxy'
+IRODS_ZONE = 'hydrotest41Zone'
+IRODS_USERNAME = 'hsproxy'
+IRODS_AUTH = 'proxywater1'
+IRODS_GLOBAL_SESSION = True
+
+# Remote user zone iRODS configuration
+REMOTE_USE_IRODS = False
+
+# iRODS customized bagit rule path
+IRODS_BAGIT_RULE = 'hydroshare/irods/ruleGenerateBagIt_HS.r'
+IRODS_BAGIT_PATH = 'bags'
+IRODS_BAGIT_POSTFIX = 'zip'
+
+IRODS_SERVICE_ACCOUNT_USERNAME = ''
+
+HS_BAGIT_README_FILE_WITH_PATH = 'docs/bagit/readme.txt'
+
+# crossref login credential for resource publication
+USE_CROSSREF_TEST = True
+CROSSREF_LOGIN_ID = ''
+CROSSREF_LOGIN_PWD = ''
+
+# Since Hyrax server on-demand update is only needed when private netCDF resources on www
+# are made public, in local development environments or VM deployments other than the www
+# production, this should not be run by setting RUN_HYRAX_UPDATE to False. RUN_HYRAX_UPDATE
+# should only be set to True on www.hydroshare.org
+RUN_HYRAX_UPDATE = False
+HYRAX_SSH_HOST = ''
+HYRAX_SSH_PROXY_USER = ''
+HYRAX_SSH_PROXY_USER_PWD = ''
+HYRAX_SCRIPT_RUN_COMMAND = ''
+
+# hsuserproxy system user configuration used to create hydroshare iRODS users on-demand
+HS_USER_ZONE_HOST = ''
+HS_USER_ZONE_PROXY_USER = ''
+HS_USER_ZONE_PROXY_USER_PWD = ''
+HS_USER_ZONE_PROXY_USER_CREATE_USER_CMD = ''
+HS_USER_ZONE_PROXY_USER_DELETE_USER_CMD = ''
+HS_USER_ZONE_PRODUCTION_PATH = '/hydroshareuserZone/home/localHydroProxy'
+
+# the local HydroShare proxy user (a counterpart of wwwHydroProxy) in a federated zone with HydroShare Zone
+HS_LOCAL_PROXY_USER_IN_FED_ZONE = 'localTestHydroProxy'
+
+# Please keep the line below unchanged since it is used to check whether
+# the current site is in production or not
+HS_WWW_IRODS_PROXY_USER = 'wwwHydroProxy'
+# credentials for HydroShare proxy user iRODS account which is set to have own access control
+# to all collections in any federated zone with HydroShare zone, which is only useful when
+# testing HydroShare federated zone in local test development environment since in www
+# production environment, IRODS_USERNAME and other associated settings already represent wwwHydroProxy settings
+HS_WWW_IRODS_PROXY_USER_PWD = ''
+HS_WWW_IRODS_HOST = ''
+HS_IRODS_LOCAL_ZONE_DEF_RES = 'hydroshareLocalResc'
+HS_WWW_IRODS_ZONE = ''
+HS_USER_IRODS_ZONE = 'hydroshareuserZone'
+
+# Email configuration
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
+HYDROSHARE_SHARED_TEMP = '/shared_tmp'
+
+# used by the mailchimp subscription job in hs_core/tasks.py
+MAILCHIMP_ACTIVE_SUBSCRIBERS = "e210a70864"
+MAILCHIMP_SUBSCRIBERS = "f0c27254e3"
+
+# sendfile support for large files
+# These must match settings in nginx
+SENDFILE_ON = False
+IRODS_USER_URI = "/irods-user"
+IRODS_DATA_URI = "/irods-data"
+LOCAL_CACHE_URI = "/local-cache"
+
+RECAPTCHA_SITE_KEY = "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
+RECAPTCHA_SECRET_KEY = "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
```

Comment:
@cuahsimarko all above I believe need to go to local_settings.py and should not be included in settings.py

Reply:
I think the pattern he is following, is default settings are placed in settings.py and then each environment can override settings in local_settings.py.  This isn't a bad direction to go in.

Reply:
@sblack-usu OK. Thanks for clarification. If that is the case, I don't think we need a local_settings.templates at all. We can just put the default configuration settings all in settings.py, then in a production environment or an environment where default settings.py does not meet the need, a non-version-controlled local_settings.py can be slated in to override the default settings.py. We can make settings.py itself be sufficient for default environment configuration to have the server up and running in a local development environment where no local_settings.py is present. @cuahsimarko is this in line with what you are thinking?

Reply:
The template is to help people. We'll learn the common settings that people typically find useful and keep them in the template. ok?

Reply:
@cuahsimarko Your change to make local_settings.py load to fail silently looks good. I get what you are doing here, and the current local_settings.template is definitely not sufficient. I don't see a need for the template since users can easily create this local_settings.py if they want to override any configurations in settings.py, using settings.py as a template. I suggest to remove local_settings.template.